Assisticant.UnitTest/NotifyDataErrorInfoTests.cs

```diff
@@ -56,14 +56,16 @@ public DateTime Birth
                 set { _death.Value = value; }
             }
 
-            public ValidationRules Rules => new ValidationRules()
-                .ForString(() => PhoneNumber, r => r
-                    .Matches(@"^[0-9\-\(\)]*$"))
+            public IValidationRules Rules => new ValidationRules()
+                .For(() => PhoneNumber)
+                    .Matches(@"^[0-9\-\(\)]*$")
```

Comment:
This `For(...)` arc is strongly-typed to string, so the `Matches(...)` method is only available for strings.